src/renderer/pages/index.vue

```diff
@@ -462,6 +462,10 @@ export default Vue.extend({
         5: 'accent',
         6: 'accent',
       }
+      this.dayColors[this.$getPrefs('meeting.mwDay')]  =
+          'secondary'
+      this.dayColors[this.$getPrefs('meeting.weDay')]  =
+          'secondary'
```

Comment:
Doesn't work for the first week, when days of the week are missing. This should fix it:

```suggestion
      if (this.currentWeek === this.$dayjs().isoWeek()) {
        this.setDayColor(this.$getPrefs('meeting.mwDay'), 'secondary')
        this.setDayColor(this.$getPrefs('meeting.weDay'), 'secondary')
      } else {
        this.dayColors[this.$getPrefs('meeting.mwDay')] = 'secondary'
        this.dayColors[this.$getPrefs('meeting.weDay')] = 'secondary'
      }
```